tests/foreman/ui/test_smartclassparameter.py

```diff
@@ -289,22 +236,25 @@ def test_positive_create_matcher_attribute_priority(
             Note - The fqdn/host should have this attribute.
         6.  Submit the change.
         7.  Go to YAML output of associated host.
+        8.  Resubmit the same form to check 1241249
 
     :expectedresults: The YAML output has the value only for fqdn matcher.
 
     :CaseLevel: Integration
 
+    :BZ: 1241249
+
     :CaseImportance: Critical
     """
     sc_param = sc_params_list.pop()
     override_value = gen_string('alphanumeric')
-    override_value2 = gen_string('alphanumeric')
+    override_value2 = "['<%= @host.domain %>', '<%= @host.mac %>']"
```

Comment:
Small query, if we hardcoding the  override_value2, will this create problem while rerunning the same test while retesting?

Reply:
while retesting there would be new parameter created, I don't think that such overrides will create a conflict